recipes/glog/all/conandata.yml

```diff
@@ -11,4 +11,3 @@ sources:
 patches:
   "0.5.0":
     - patch_file: "patches/0001-fix-msvc-snprintf.patch"
```

Comment:
```suggestion
    - patch_file: "patches/0001-fix-msvc-snprintf.patch"
      patch_description: "Use stdio.h instead of cstdio.h for snprintf"
      patch_type: "conan"
```